package-lock.json

```diff
@@ -0,0 +1,3657 @@
+{
```

Comment:
By convention, we have chosen not to commit package-lock.json files to our repositories.  I'd suggest adding it to the .gitignore and ensuring that it's no longer tracked as well.